class ValidationError(Exception):
    pass


class ObjectAlreadyExists(Exception):
    pass